docs/error-messages/compiler-errors-1/compiler-error-c2390.md

```diff
@@ -22,7 +23,7 @@ Possible resolutions:
 
 ## Example
 
-- The following sample generates C2390:
+- The following example generates C2390:
```

Comment:
A separate PR will be issued to remove the peculiar bullet point once this PR is merged.